pkg/config/config.go

```diff
@@ -149,3 +150,15 @@ func GetRayClusters(ctx context.Context, c client.Client) ([]RayCluster, error)
 	}
 	return config.RayClusters, nil
 }
+
+func GetDefaultRerank(ctx context.Context, c client.Client) (*arcadiav1alpha1.TypedObjectReference, error) {
```

Comment:
Which default reranking model do you prefer based on your tests?  We can update our config and post-models.yaml in helm chart